css/css-values/attr-in-max.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>
```

Comment:
Can you compact the markup up a bit? The "INDENT TABS" lint rule is disabled for this directory and there's no rules for indentation, but the more compact style of https://github.com/web-platform-tests/wpt/commit/076ec518433677b21cda600195208f60c83dee39 or 07c52e79870a5c7cf7aa37a7bc2c3e232b295d03 (the two most recent commits in this directory) would be great!

Reply:
@foolip Just pushed a commit to compact the markup. Thanks!